### ISSUE TITLE
data.set( hash ) empties hash

### ISSUE THREAD

**Issue by kuraga** [arqex/freezer#24]
Is it normal?
```javascript
var store = new Freezer({ time: 0 });
var data = store.get();

var changes = { time: 1 };
data.set(changes);
console.log(changes); // {}
// changes is empty object now!
```

**Comment by arqex** [comment]
Nice catch!

I will have a look at it.

**Comment by arqex** [comment]
Hi @kuraga 
This has been fixed in v0.5.2 thanks for your help!
